### DISCUSSION TITLE
qtable and separators, pixel problem

### DISCUSSION THREAD

**Discussion by AmosSzust** [quasarframework/quasar#16982] [General - Components / Directives / etc]
I use qtable with `separator="horizontal"` and I added the following css to make it white and also a sticky first column
```
.q-table__middle .q-table tbody td {
  border-color: white;
}

.my-sticky-header-column-table {
  max-height: 1310px;
}
.my-sticky-header-column-table td:first-child {
  background-color: rgba(4, 52, 184, 0.99);
  color: white;
}
.my-sticky-header-column-table tr th {
  position: sticky;
  z-index: 2;
  background: rgba(5, 62, 177, 0.99);
}
.my-sticky-header-column-table thead tr:last-child th {
  top: 48px;
  z-index: 3;
}
.my-sticky-header-column-table thead tr:first-child th {
  top: 0;
  z-index: 1;
}
.my-sticky-header-column-table tr:first-child th:first-child {
  z-index: 3;
}
.my-sticky-header-column-table td:first-child {
  z-index: 1;
}
.my-sticky-header-column-table td:first-child,
.my-sticky-header-column-table th:first-child {
  position: sticky;
  left: 0;
}
.my-sticky-header-column-table tbody tr:last-child {
  bottom: 0;
  position: sticky;
  z-index: 1;
  background-color: rgba(4, 52, 184, 0.99);
}
```

It works great except for the fact that every other row, the separator is 1 pixel above in the fixed column, comparing to the rest of it
![image](https://github.com/quasarframework/quasar/assets/100950035/1ac23909-5efa-4497-91e4-d6449c16264b)

What am I doing wrong? Is there a better way to make the separator white? Thanks

I managed to reproduce it in codepen, notice the 1st and 3rd cells. If I remove `dense` then it's ok but I want it.
https://codepen.io/amosss/pen/abxNbye

**Comment by AmosSzust** [comment]
Still struggling with it, did a codepen. I hope someone here will know what to do and share his thoughts, thanks :)

**Comment by AmosSzust** [comment]
One more reminder, https://codepen.io/amosss/pen/abxNbye
maybe this  time :)

**Reply by kenyon-grublr** [reply]
Apologies for this 'non-answer' to your specific issue. But did you check out the example for just using the q-table [here](https://quasar.dev/vue-components/table#example--sticky-first-column)](https://quasar.dev/vue-components/table#example--sticky-first-column)

Looks like they were able to apply the sticky class to the first column pretty easily. Then you would just need to work on a style override for the separator.

If using the q-markup-table is a must then perhaps the style class in the example will still help you.

**Reply by AmosSzust** [reply]
I don't remember when I first saw this problem, meaning what exactly did I add/do that made this glitch visible. It took me a while to prepare the codepen hoping it will bubble up the problem faster to the people who are more css oriented.
